ec2-cargo/src/main.rs

```diff
@@ -45,7 +45,10 @@ async fn main() {
         return;
     }
 
-    let aws = Aws::builder(CleanupResources::AllResources).build().await;
+    let aws = Aws::builder(CleanupResources::AllResources)
+        .use_az(Some("us-east-1b".into()))
```

Comment:
why us-east-1b? I thought us-east-1c was the default.

Reply:
us-east-1c lacks a default subnet in our internal environment, therefore we are using us-east-1b for shotover infra.

Reply:
fair enough. In that case, should the default in https://github.com/shotover/aws-throwaway/blob/main/aws-throwaway/src/backend/sdk/aws.rs#L59 be us-east-1b as well?

Reply:
I kept it the same there for backwards compatibility in case someone else is using the library.